README.md

```diff
@@ -22,50 +22,118 @@ although basic usage is shown below.
 
 ### User Registration Flow
 
+##### Server endpoint (registration challenge):
+
 ```javascript
-// 1. Check that the user is logged in.
+const u2f = require('u2f');
 
-// 2. (On server) generate user registration request and save it in session:
-var u2f = require('u2f');
+function handler(req, res) {
+  // 1. Check that the user is logged in.
 
-var req = u2f.request("<appId>");
-session.authRequest = req;
+  // 2. Generate a registration request and save it in the session.
+  // The app ID is usually the fully qualified URL of your website. The website MUST be HTTPS, or
```

Comment:
Maybe move the app_id to be a global const for the app and move this comment there?